samples/subsys/ipm/ipm.rst

```diff
@@ -0,0 +1,10 @@
+.. _ipm_samples:
```

Comment:
ipm is a driver in Zephyr. I think these samples shall be placed in `samples/drivers/ipm` instead of `samples/subsys`

Reply:
yeah, not sure why they were in `subsys/ipc` in the first place to be honest.